core/src/com/unciv/ui/pickerscreens/TechPickerScreen.kt

```diff
@@ -95,30 +96,30 @@ class TechPickerScreen(internal val civInfo: CivilizationInfo, centerOnTech: Tec
         val allTechs = civInfo.gameInfo.ruleSet.technologies.values
         if (allTechs.isEmpty()) return
         val columns = allTechs.map { it.column!!.columnNumber }.maxOrNull()!! + 1
-        val rows = allTechs.map { it.row }.maxOrNull()!! + 1
+        val rows = allTechs.map { it.row }.maxOrNull()!!
```

Comment:
I'm almost certain that removing the +1 will lead to a crash - since the techMatrix will not be arrayOfNulls(allTechs.map { it.row }.maxOrNull()!!) and thus val tech = techMatrix[columnIndex][rowIndex] on line 125 will fail...

Reply:
I test it's ok, because in fact row-index  starts from 1 and columns-index starts from 0. 
They are different.